Sources/MongoSwift/ChangeStream.swift

```diff
@@ -3,12 +3,17 @@ import mongoc
 /// `ChangeStreamOptions` to resume or start a change stream where a previous one left off.
 /// - SeeAlso: https://docs.mongodb.com/manual/changeStreams/#resume-a-change-stream
 public struct ResumeToken: Codable, Equatable {
-    private let resumeToken: Document
+    public let resumeToken: Document
```

Comment:
why did this become public?

Reply:
oops, I made that public while trying to debug. I've made it private now.